src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -85,4 +92,9 @@ public GuiSettings getGuiSettings() {
     public void setGuiSettings(GuiSettings guiSettings) {
         model.setGuiSettings(guiSettings);
     }
+
+    @Override
+    public Person getLastViewedPerson() {
+        return lastViewedPerson;
```

Comment:
As mentioned earlier, can we keep this as a static variable rather than LogicManager's member variable?